modules/swift_argument_parser/1.3.0/presubmit.yml

```
@@ -0,0 +1,8 @@
+tasks:
+  verify_targets:
+    name: Verify build targets
+    platform: macos
```

```suggestion
    platform: macos
    bazel: 7.x
```

looks like a new requirement